README.md

```diff
@@ -10,8 +14,57 @@
 
 
 ## Usage
-* Tests must end in `_(spec|test).(coffee|js)`
-* Run tests with ```mycha run```
+
+* `mycha install <testHelperPath>`
```

Comment:
Nitpicking here, but it should probably query the test helper path from the user. This command is run infrequent enough that people don't remember its syntax.